src/components/Upgrade/UpgradeBanner.js

```diff
@@ -27,13 +27,14 @@ function useUpgradeBanner(onUpgrade) {
 
   useEffect(() => {
     if (
+      visible &&
```

Comment:
This is not going to work when we want to want to open the showcase modal from Organization but let's merge this and let me fix it in https://github.com/aragon/aragon/pull/949.